config/environments/production.rb

```diff
@@ -26,6 +26,9 @@
   config.assets.js_compressor = :uglifier
   # config.assets.css_compressor = :sass
 
+  # Set a css_compressor so sassc-rails does not overwrite the compressor when running the tests
```

Comment:
This isn't related to tests is it?

Reply:
I thought it was but looking again, I think it is the asset compile in production that causes it?

Reply:
Ah ok. Makes sense.